### PR TITLE
Prompt on empty input (create/remove), misc. golint improvements

### PR DESCRIPTION
**Pull request by alvinantonius**
- add improvement on create and remove command so user can add the url later if they forgot the flag
- on remove command, user can remove multiple repo as args to the command using space as delimiter
- changes some code so they pass the golint checking.

### CODE REVIEW

File: cmd/remove.go [hunk 1]
```diff
@@ -45,5 +50,19 @@ func findRepo(repoName string) (entry, error) {
...
+}
+
+func removeRepo(repoName string) error {
+	repo, err := findRepo(repoName)
+	if err != nil {
+		return fmt.Errorf("repository `%s` not found", repoName)
+	}
+
+	_, err = client.Repositories.Delete(ctx, repo.Owner, repo.Repo)
+	checkError(err)
```
Comment (kashav) [inline cmd/remove.go]: I think it'd be better if this function returned the error and we checked the error in `Run`

```go
if err != nil {
    return err
}
```

File: cmd/remove.go [hunk 2]
```diff
@@ -17,18 +17,23 @@ Example:
...
+			fmt.Scanln(&repoName)
+			args = append(args, repoName)
 		}
 
-		_, err = client.Repositories.Delete(ctx, repo.Owner, repo.Repo)
-		checkError(err)
+		// then do the repo removal
+		for _, arg := range args {
+			err := removeRepo(arg)
+			if err != nil {
```
Comment (kashav) [inline cmd/remove.go]: We can just use `checkError` here.

File: cmd/remove.go [hunk 3]
```diff
@@ -45,5 +50,19 @@ func findRepo(repoName string) (entry, error) {
...
 	}
 
-	return entry{}, errors.New("Repository not found.")
+	return entry{}, errors.New("repository not found")
+}
+
+func removeRepo(repoName string) error {
+	repo, err := findRepo(repoName)
+	if err != nil {
+		return fmt.Errorf("repository `%s` not found", repoName)
```
Comment (kashav) [inline cmd/remove.go]: `fmt.Errorf` 👍 

Can we change the message to `"Repository '%s' not found."` (backticks around the `%s`)

File: cmd/remove.go [hunk 4]
```diff
@@ -17,18 +17,23 @@ Example:
...
+			var repoName string
+			fmt.Print("Please input shortened URL you want to remove : ")
+			fmt.Scanln(&repoName)
+			args = append(args, repoName)
 		}
 
-		_, err = client.Repositories.Delete(ctx, repo.Owner, repo.Repo)
-		checkError(err)
+		// then do the repo removal
+		for _, arg := range args {
```
Comment (kashav) [inline cmd/remove.go:34]: Nice! Can we get this added this to the [`remove` help dialogue](https://github.com/kshvmdn/shorten/blob/master/cmd/remove.go#L13) (and an example too)? Just so users know that they can optionally provide a space-separated list of repos to remove.

**Review by kashav — CHANGES_REQUESTED** [review]

**Comment by kashav** [comment]
@alvinantonius Looks good! Found a few things that can be changed - let me know what you think!

File: cmd/remove.go [hunk 5]
```diff
@@ -17,18 +17,23 @@ Example:
...
 
-		repo, err := findRepo(repoName)
-		if err != nil {
-			fmt.Printf("Repository `%s` not found.\n", repoName)
-			return
+		// do input check before running command
+		if len(args) == 0 {
+			var repoName string
+			fmt.Print("Please input shortened URL you want to remove : ")
+			fmt.Scanln(&repoName)
```
Comment (kashav) [inline cmd/remove.go]: We should also verify here that `repoName` is not `""`

File: cmd/create.go [hunk 6]
```diff
@@ -36,7 +36,8 @@ Example:
   $ shorten create --url=https://github.com/golang/go --name=go --private`,
 	Run: func(cmd *cobra.Command, args []string) {
 		if url == "" {
-			log.Fatal("Expected URL flag.")
+			fmt.Print("Please input URL you want to shorten : ")
+			fmt.Scanln(&url)
```
Comment (kashav) [inline cmd/create.go]: Same as below (check that `url` is not `""`).

File: cmd/remove.go [hunk 7]
```diff
@@ -17,18 +17,23 @@ Example:
...
-		repoName := args[0]
 
-		repo, err := findRepo(repoName)
-		if err != nil {
-			fmt.Printf("Repository `%s` not found.\n", repoName)
-			return
+		// do input check before running command
+		if len(args) == 0 {
+			var repoName string
+			fmt.Print("Please input shortened URL you want to remove : ")
```
Comment (kashav) [inline cmd/remove.go]: This should read:

`"Please input the name(s) of the repos you'd like to remove: "`

File: cmd/remove.go (hunk 3 above)
Comment (alvinantonius) [reply cmd/remove.go]: will do

File: cmd/remove.go (hunk 5 above)
Comment (alvinantonius) [reply cmd/remove.go]: i'm thinking to return log.Fatal if the string is empty.
what do you think about it?

File: cmd/remove.go (hunk 4 above)
Comment (alvinantonius) [reply cmd/remove.go:34]: i missed that one,
will update the help dialogue soon

File: cmd/remove.go (hunk 5 above)
Comment (kshvmdn) [inline cmd/remove.go]: Sounds good!

**Comment by alvinantonius** [comment]
Already did some changes.
please check sir.

thanks in advance

File: cmd/remove.go [hunk 8]
```diff
@@ -11,27 +11,29 @@ var removeCmd = &cobra.Command{
...
 Example:
   $ shorten remove go
-  $ shorten remove x29jzI8m`,
+  $ shorten remove x29jzI8m
+  $ shorten remove a243234a jasdH234 524aAdsd`,
 	Run: func(cmd *cobra.Command, args []string) {
 
 		// do input check before running command
-		if len(args) == 0 {
+		for len(args) == 0 {
```
Comment (kashav) [inline cmd/remove.go]: Nice!

**Review by kashav — APPROVED** [review]

**Comment by kashav** [comment]
Looks good to me! Thanks @alvinantonius!

**Pull request merged**